app/app.go

```diff
@@ -359,10 +359,19 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	/* New Module Wiring START */
 	newModuleManager := servermodule.NewManager(app.BaseApp, codec.NewProtoCodec(interfaceRegistry))
 
-	err := newModuleManager.RegisterModules(NewModules)
+	// BEGIN HACK: this is a total, ugly hack until x/auth supports ADR 033 or we have a suitable alternative
```

Comment:
Is this a hack or a TODO? 

Why we can't use a `MsgServer` client here?

Reply:
It's a brutal hack

Reply:
x/auth has no MsgServer